### DISCUSSION TITLE
Adding custom EPDG Server address for WiFi Calling (Globe Telecom - Philippines)

### DISCUSSION THREAD

**Discussion by Squiggly9461** [kyujin-cho/pixel-volte-patch#172] [Carrier-specific Experiences]
My carrier Globe Telecom (Philippines) has a carrier profile enabled by default on Pixel with carrier profile globe_ph however it only enabled VoLTE. 

Now I'm trying my luck with Pixel IMS to have WiFi Calling enabled however Globe Telecom uses a custom EPDG Address for their WiFi Calling rather than the standard 3gpp address. 

Globe Telecommunications uses the following EPDG:

`weconnect.globe.com.ph `

is there anyway to add this custom address on the PixelIMS app so we can use WIFI Calling? 

Thanks!

**Comment by Squiggly9461** [comment]
@kyujin-cho hopefully you can check this and see if its possible. Thanks!

**Comment by flyingcards** [comment]
I have the same problem with my mobile provider Telenet in Belgium.
